### PR TITLE
Fix output of diff to work on python3

### PR DESCRIPTION
**Pull request by rhatdan**
rpm returns bytes for python3, need to fix encoding to printout correctly.

### CODE REVIEW

**Comment by rhatdan** [comment]
@baude @cgwalters PTAL

File: Atomic/diff.py [hunk 1]
```diff
@@ -158,16 +158,18 @@ def _get_rpms(self, chroot_os):
         ts = rpm.TransactionSet(chroot_os)
         ts.setVSFlags((rpm._RPMVSF_NOSIGNATURES | rpm._RPMVSF_NODIGESTS))
         image_rpms = []
+        enc=sys.getdefaultencoding()
         for hdr in ts.dbMatch():  # No sorting  # pylint: disable=no-member
             if hdr['name'] == 'gpg-pubkey':
```
Comment (baude) [inline Atomic/diff.py]: Does this need to be decoded to compare properly? similar to the mount issue ?

Comment (rhatdan) [inline Atomic/diff.py]: Yes good catch

Comment (rhatdan) [inline Atomic/diff.py]: Fixed.

**Comment by baude** [comment]
Looks great @rhatdan

**Pull request merged**